sass/includes/search/_search-results-hero.scss

```diff
@@ -118,17 +121,17 @@
             height: 100%;
             padding: 0.5rem;
             text-decoration: none;
-            color: #000000; // IE11
+            color: #000; // IE11
```

Comment:
Here you can see that the option for duplicate properties applied in `.stylelintrc` is working as expected

Reply:
These look like semi-transparent colour values? If they are, could we change these to a single `rgba()` definitions which work in IE11?

Reply:
Done - turned out this was the only one for ie11 - all the others the linter picked up were duplicates that should not have been there - so that rule is reinstated.